examples/workflows/volume-mounts.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     script:
       command:
       - python
-      image: python:3.7
+      image: python:3.8
```

Comment:
Changed default python version to 3.8 as that's the minimum required version for hera.